hms-app/src/components/card-details/HackathonDetails.tsx

```diff
@@ -231,14 +249,18 @@ export default function HackathonDetails(props: IProps) {
         </div>
       )}
 
-      {hackathonData.startDate !== NULL_DATE &&
-        hackathonData.startDate.toString() !== 'Invalid Date' &&
+      {hackathonData.startDate !== NULL_DATE && hackathonData.startDate.toString() !== 'Invalid Date' &&
         !isHackathonLoading &&
         !isHackathonError &&
         (type === HackathonDetailsType.Header ||
           type === HackathonDetailsType.Archive) && (
           <div>
-            <HackathonHeader hackathonData={hackathonData} />
+            <h2>Title: {hackathonData.title}</h2>
+            <h2>
+              Start Date: {new Date(hackathonData.startDate).toDateString()} End
+              Date: {new Date(hackathonData.endDate).toDateString()}
+            </h2>
+            <h2>All Ideas ({hackathonData.ideas?.length})</h2>
```

Comment:
why have you deleted the hackathonheader component? looks like an old version with this h2`s